recipes/adept/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "Adept" %}
+{% set version = "1.1a" %}
+{% set build_number = "0" %}
+{% set sha256 = "fbe3e4d032bddaf361f1c2fe40263d802da0f4752dfe385a591d247a6d0f6627" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/arghdos/Adept/releases/download/v1.1a/Adept-v11a.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build_number }}
+  skip: True  # [not unix]
+
+requirements:
+  build:
+    - toolchain
+    - autoconf
+    - automake
+    - libtool
+  run:
+    - libgcc
+
+test:
+  commands:
+    - test -L $PREFIX/lib/libadept.so  # [linux]
+    - test -L $PREFIX/lib/libadept.dylib  # [osx]
+    - test -f $PREFIX/lib/libadept.so.0.0.0  # [linux]
+    - test -f $PREFIX/lib/libadept.a  # [linux]
+    - test -f $PREFIX/lib/libadept.la  # [osx]
+    - test -f $PREFIX/lib/libadept.0.dylib  # [osx]
+    - test -f $PREFIX/include/adept.h  # [linux or osx]
+
+about:
+  home: http://www.met.reading.ac.uk/clouds/adept/
+  license: Apache-2.0
+  license_family: Apache
+  license_file: COPYING
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
```

Comment:
This looks like a left over from c&p :)

Reply:
Whoops, fixed.  The license type and file were correct, just updated the summary